ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -606,3 +606,33 @@ def test_jax_numpy_einsum(
         _use_xeinsum=False,
         **kw
     )
+
+
+# subtract
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), num_arrays=2, shared_dtype=True
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.subtract"
+    ),
+)
+def test_jax_numpy_subtract(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="jax",
+        fn_tree="numpy.subtract",
+        x1=x[0],
+        x2=x[0],
```

Comment:
this should be x[1] right?

Reply:
Ok, fixed x2=x[1]

Reply:
The pull request has been closed without merge. Any issue with my pull request?